components/layout/SectionContainer.tsx

```diff
@@ -11,7 +11,8 @@ export default function SectionContainer({
       component="section"
       id={id}
       maxWidth="lg"
-      sx={{ my: 8, scrollMarginTop: 64 }}
+      disableGutters
```

Comment:
Added this so we can pass px prop to sectionContainer. Otherwise the passed px would have been overridden.